src/send/SendConfirmation.tsx

```diff
@@ -152,6 +152,7 @@ function SendConfirmation(props: Props) {
         <LineItemRow
           testID="SendConfirmation/fee"
           title={t('feeEstimate')}
+          textStyle={{ ...typeScale.bodyMedium }}
```

Comment:
Why is this being added? Is it different from what it was defaulting to or just ensuring that it is this?

Reply:
It defaults to `fontStyles.regular`. This was closer to the mocks in regards to font size.

Reply:
does this need a color too? I don't think the new styles default a color like the old ones.

Reply:
`src/components/LineItemRow.tsx` has a default text color set to `Colors.black` and the subtext is set to `colors.gray4`.